scripts/install-stable.sh

```diff
@@ -57,34 +57,83 @@ else
     tar -xzvf apache-maven-${mavenVersion}-bin.tar.gz
 fi
 
-# Download Ozone
-echo "$INFO Downloading Ozone $ozoneVersion..."
-$mvn dependency:get \
--DgroupId=com.ozonehis \
--DartifactId=ozone \
--Dversion=${ozoneVersion} \
--Dpackaging=zip \
--DremoteRepositories=https://nexus.mekomsolutions.net/repository/maven-public \
--Dtransitive=false
-
-# Copy and Unpack
-echo "$INFO Extracting Ozone..."
-$mvn dependency:copy \
--Dartifact=com.ozonehis:ozone:${ozoneVersion}:zip \
--DoutputDirectory=./ \
--DuseBaseVersion=true
-
-# Account for some inconsistencies in some systems where Maven copies the files without a timestamp.
-# Only rename the file if it is time-stamped.
-if [ ! -f "ozone-${ozoneVersion}.zip" ]
-then
-    echo "$INFO Rename to 'ozone-${ozoneVersion}.zip'..."
-    mv ozone-1.0.0-*.zip ozone-${ozoneVersion}.zip
-fi
+# Set up local Maven helper project to workaround issues when using the Maven dependency plugin from CLI.
+cat >_temp_install-latest-ozone-pom.xml <<EOF
```

Comment:
Seems like we should not name the `pom.xml` to `_temp_install-latest-ozone-pom.xml` as maven is not able to recognise it when building the project.

Here is why:
Got the below error when testing the PR
![Screenshot 2024-06-28 at 3 07 54 PM](https://github.com/ozone-his/ozone/assets/36266597/f32a2c81-e76a-4fad-8882-bc45353a7ca1)

When I renamed the `_temp_install-latest-ozone-pom.xml` ->  `pom.xml` it worked fine.
cc: @rbuisson

Reply:
An oversight.
The mvn command should be `mvn clean install -f _temp_install-latest-ozone-pom.xml`.
Let me fix it.

Reply:
Sure